docs/decisions/0003-reduce-bokchoy-testing.rst

```diff
@@ -0,0 +1,68 @@
+Status
+======
+Draft
+
+
+Context
+=======
+
+edx-platform bokchoy tests are slow, flaky and difficult to debug.  A quick assessment of their value shows that they might be more trouble than they are worth.  And that we might get the same benefit with far fewer tests.
+
+Baseline Data:
+--------------
+
+This data was collected based on the results of bokchoy tests run across all edx-platform PRs over the last 7 days.
+
+* Total number of builds: 253(across 106 PRs)
+* Failures: 49(across 24 PRs)
+
+  * True Failures: 10(across 6 PRs)
+  * Failures that wouldn’t be caught by other test: 3(on 1 PR)
+
+Color
+~~~~~
+
+Of the real failures found, there was one PR which had a failure that was only found via bokchoy and a11y tests.
+    * This PR made a JS change which would have broken many pages from loading.
+
+Recommendation
```

Comment:
Are you intentionally deviating from the standard ADR structure of having a `Decision` section?  See https://github.com/edx/credentials/blob/master/docs/decisions/0001-record-architecture-decisions.rst

Reply:
Yea, my plan was to add that after the experiment has run it's course. Since we won't have made a decision until then.

Reply:
Ok.